app/facades/endgame_facade.rb

```diff
@@ -1,15 +1,16 @@
 class EndgameFacade 
   def self.get_phrase_and_gif(wins, rounds)
-    percentage = score_percentage(wins, rounds)
-    type = find_phrase_type(percentage)
+    perc  = score_percentage(wins, rounds)
+    type  = find_phrase_type(perc)
     phrase = get_endgame_phrase(type)
```

Comment:
in endgame_controller, indentations all line up (equal signs are aligned". In endgame_facade, they are staggered based on length of variable. Let's pick a convention and adhere to it throughout. I vote for everything in line )as in endgame_controller). 
Also, I think 'perc' is a vague descriptor, percentage is long but there is no misunderstanding it.

Reply:
agreed. going with "percent"